### DISCUSSION TITLE
s/mzeux4mnxkyiwzwdwhkboq

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#22] [Announcements]
# s/mzeux4mnxkyiwzwdwhkboq

以数据的方式告诉你财务自由提前退休并没有那么难

https://blog.ops-coffee.cn/s/mzeux4mnxkyiwzwdwhkboq

**Comment by thetbw** [comment]
每年增加一万对于普通人可不容易，这应该是个抛物线，后面其实是每年在减少的，因为要花钱的地方越来越多了，就算没有小孩也要考虑父母
